### DISCUSSION TITLE
just a quick note

### DISCUSSION THREAD

**Discussion by kovacsarona** [Luwx/Lightly#49] [Ideas]
Hi
I really like Lightly and use it so thanks for making it.
Could you make a mixed style between fluent(kv)and Lightly which integrated even better with Breeze style

also Could you make something that enables transparent colour scheme support in Breeze too? (n0000b question)

**Comment by Luwx** [comment]
You can get pretty close to the fluent kvantum theme with Lightly, specially the light version https://github.com/Luwx/Fluent-kvantum/blob/master/Fluent-light.png, you'll have to use a completely transparent color scheme and reduce the corner radius. You can also disable the shadows.

> also Could you make something that enables transparent colour scheme support in Breeze too? 

That's not possible without patching breeze :/
